applications/ecallmgr/src/ecallmgr_config.erl

```diff
@@ -30,7 +30,7 @@
 -include_lib("kazoo/include/kz_databases.hrl").
 
 -spec flush() -> 'ok'.
--spec flush(kz_json:key()) -> 'ok' | {'error', any()}.
+-spec flush(kz_json:keys()) -> 'ok' | {'error', any()}.
```

Comment:
This type name change (`key()` -> `keys()`) in `kz_json` was really unfortunate IMO: now any function taking a `JObj`'s "key path" must accept a plural "`keys()`", which is very misleading. Connotation between singular path & plural keys will lose more than one newcomer.

Also, now functions that do take a list of "key paths" have to use `[ keys() ]`...
Why not have stuck with `key() :: [json_string()] | json_string()`?
We know that a "key path" can be either a json_string or a list of these. The name of the "key path" type does not need to reflect some type in its union definition is a list, does it?